app/controllers/admin/project_groups_controller.rb

```diff
@@ -14,7 +19,13 @@ def create
     authorize @entry
 
     @loan = @entry.project
-    @entry.parent = @loan.root_timeline_entry
+
+    parent_id = project_group_params[:parent_id]
+    if parent_id && parent_id.present?
+      @entry.parent = ProjectGroup.find(parent_id)
+    else
+      @entry.parent = @loan.root_timeline_entry
+    end
```

Comment:
This should be just `@entry.parent = parent_id.present? ? ProjectGroup.find(parent_id) : @loan.root_timeline_entry` I think.